nixos/modules/system/boot/kexec.nix

```diff
@@ -13,6 +13,9 @@
         path = [ pkgs.kexectools ];
         script =
           ''
+            # Don't load the current system profile if we already have a kernel loaded
+            [[ 1 = "$(</sys/kernel/kexec_loaded)" ]] && exit
```

Comment:
I think it would be good to print something in both cases (kexec already loaded or not), so that the user knows what's going on.

(I say that coming from debugging kexec issues for > 30 hours this weekend.)

Reply:
Where to?

Reply:
I was thinking just stdout, so that it appears in `journalctl`.